lib/requester/core.js

```diff
@@ -76,6 +76,26 @@ var dns = require('dns'),
         removeRefererHeader: 'removeRefererHeaderOnRedirect'
     },
 
+    /**
+     * Helper function to extract top level domain for the given hostname
+     *
+     * @private
+     *
+     * @param hostname
+     * @return string
+     */
+    getTLD = function (hostname) {
+        if (!hostname) {
+            return '';
+        }
+
+        hostname = String(hostname);
+
+        var index = hostname.lastIndexOf('.');
+
+        return hostname.substring(index + 1);
```

Comment:
```suggestion
        return hostname.substring(hostname.lastIndexOf('.') + 1);
```
Because `index` is not required anywhere else.